build/constants.yaml

```diff
@@ -47,29 +47,29 @@
     dot-net-60-sdk-version: 6.0.417
     dot-net-60-rtm-sdk-version: 6-0-100-rtm-21527-11
     dot-net-70-sdk-version: 7.0.404
-    dot-net-80-sdk-version: 8.0.201
+    dot-net-80-sdk-version: 8.0.202
     runtime-versions:
-      - 3.0-debian-buster
-      - 3.1-debian-bullseye
-      - 3.1-debian-buster
-      - 5.0-debian-buster
-      - 6.0-debian-bullseye
-      - 6.0-debian-buster
-      - 7.0-debian-bullseye
-      - 7.0-debian-buster
-      - 8.0-debian-bullseye
-      - 8.0-debian-bookworm
-      - dynamic-debian-buster
+    - 3.0-debian-buster
+    - 3.1-debian-bullseye
+    - 3.1-debian-buster
+    - 5.0-debian-buster
+    - 6.0-debian-bullseye
+    - 6.0-debian-buster
+    - 7.0-debian-bullseye
+    - 7.0-debian-buster
+    - 8.0-debian-bullseye
+    - 8.0-debian-bookworm
+    - dynamic-debian-buster
```

Comment:
Let's get clarity on these dynamic images also for the future

Reply:
Sure sure..